subsys/zbus/zbus.c

```diff
@@ -44,17 +81,33 @@ int _zbus_init(void)
 SYS_INIT(_zbus_init, APPLICATION, CONFIG_ZBUS_CHANNELS_SYS_INIT_PRIORITY);
 
 static inline int _zbus_notify_observer(const struct zbus_channel *chan,
-					const struct zbus_observer *obs, k_timepoint_t end_time)
+					const struct zbus_observer *obs, k_timepoint_t end_time,
+					struct net_buf *buf)
 {
 	int err = 0;
 
-	if (obs->type == ZBUS_OBSERVER_LISTENER_TYPE) {
+	switch (obs->type) {
+	case ZBUS_OBSERVER_LISTENER_TYPE: {
```

Comment:
You don't need braces on the case statement.

Reply:
I guess there's no rule about that but at least keep the `break` inside the braced block, then at least it'd be easy to spot stuff like the

```c
return ...;
} break;
```

 below

Reply:
Done.